docs/features/software-templates/writing-custom-actions.md

```diff
@@ -217,6 +219,28 @@ import {
     })
 ```
 
+### Using Checkpoints in Custom Actions (Experimental)
+
+Idempotent action could be achieved via the usage of checkpoints.
+
+Example:
+
+```ts
```

Comment:
Could you add the path to the file where this code is usually added? Something like:
`ts title="path/to/the/file.ext"`